modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,219 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
 ======
 =====
 
+=== Access Neo4j using a Kubernetes Ingress
+
+The Neo4j Helm charts provide a helm chart that allows you to use a Kubernetes Ingress to access Neo4j running on port _:80_ or _:443_.
+The helm chart is called _neo4j/neo4j-reverse-proxy_ and is available on the _neo4j_ helm repository from version 5.12.0.
+For more information about Kubernetes Ingress, see the link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes official documentation -> Ingress].
+
+The helm chart creates a reverse proxy that is configured to route traffic to the Neo4j service URL using the `serviceName`, `namespace`, and `domain` values.
+For example, if the `serviceName` is _standalone-admin_, the `namespace` is _default_, and the `domain` is _cluster.local_, then the Neo4j service URL is _standalone-admin.default.svc.cluster.local_.
+
+image::reverse-proxy.svg[title="Reverse proxy flow diagram"]
```

Comment:
We should also mention what the reverse proxy this which is the most important thing missing from here...
We should mention that the reverse proxy creates a http server which based on the request headers routes request to either bolt reverse proxy or http reverse proxy. Upon receiving a response the bolt reverse proxy updates the response to replace the bolt port with either 80 or 443